packages/boxel/package.json

```diff
@@ -149,7 +154,7 @@
     "qunit-dom": "^2.0.0",
     "stylelint": "^14.3.0",
     "stylelint-config-standard": "^24.0.0",
-    "typescript": "^4.2.3",
+    "typescript": "^4.7.4",
```

Comment:
This was the source of the `f.createClassStaticBlockDeclaration` declaration. I guessed babel version problem originally, but it was actually typescript version problem.